client/src/Components/HistoryModalContent/HistoryModalContent.jsx

```diff
@@ -1,25 +1,45 @@
-import React, { forwardRef } from 'react';
-import { Box, Typography, IconButton, Container } from '@material-ui/core';
+import React, { forwardRef, Fragment } from 'react';
+import {
+  IconButton,
+  Table,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TableBody,
+  Box,
+  DialogContent,
+  DialogTitle,
+} from '@material-ui/core';
 import CloseIcon from '@material-ui/icons/Close';
 
-import MainBox from 'Components/MainBox';
-import historyModalContentStyles from './HistoryModalContentStyles';
-import { withStyles } from '@material-ui/styles';
+import { historyModalContentStyles, HeadTableCell } from './HistoryModalContentStyles';
+
+const HistoryModalContent = ({ children, onClose, modalTitle }) => {
+  const classes = historyModalContentStyles();
 
-const HistoryModalContent = withStyles(historyModalContentStyles)(({ forwardedRef, children, onClose, classes }) => {
   return (
-    <Container maxWidth="md" className={classes.root}>
-      <MainBox innerRef={forwardedRef} padding={2}>
-        <IconButton aria-label="close" className={classes.closeButton} onClick={onClose}>
-          <CloseIcon />
-        </IconButton>
-        <Typography variant="h6" component="h2" gutterBottom>
-          History
-        </Typography>
-        <Box>{children}</Box>
-      </MainBox>
-    </Container>
+    <Fragment>

```

Comment:
zamiast używania `<Fragment></Fragment>` można zapisać po prostu `<></>` React rozumie to tak samo a mniej pisania :)